test/kubemark/start-kubemark.sh

```diff
@@ -97,7 +138,8 @@ gcloud compute ssh --zone=${ZONE} --project="${PROJECT}" ${MASTER_NAME} \
   sudo bash -c \"echo \"${KUBE_BEARER_TOKEN},admin,admin\" > /srv/kubernetes/known_tokens.csv\" && \
   sudo bash -c \"echo \"${KUBELET_TOKEN},kubelet,kubelet\" >> /srv/kubernetes/known_tokens.csv\" && \
   sudo bash -c \"echo \"${KUBE_PROXY_TOKEN},kube_proxy,kube_proxy\" >> /srv/kubernetes/known_tokens.csv\" && \
-  sudo bash -c \"echo admin,admin,admin > /srv/kubernetes/basic_auth.csv\""
+  sudo bash -c \"echo admin,admin,admin > /srv/kubernetes/basic_auth.csv\" && \
+  sudo bash -c \"echo -n ${EVENT_STORE_IP:-127.0.0.1} > /tmp/event-store-ip\""
```

Comment:
I don't have any better idea. I can either set it directly in .bashrc-ish thing, use salt (or sth), or pass it directly to the bash command running start-kubemark-master.sh (which makes it more unreadable in my opinion).

If you prefer me to do it in one of above ways (short of salt - that I won't do), or have some other idea let me know.